test/test_clean_element.rb

```diff
@@ -0,0 +1,190 @@
+# encoding: utf-8
+require_relative 'common'
+
+describe 'Sanitize::Transformers::CleanElement' do
+  make_my_diffs_pretty!
+  parallelize_me!
+
+  strings = {
```

Comment:
These test outputs are duplicates of the ones in `test_sanitize.rb`. They're also not actually being used in this file.